DQM/SiStripMonitorClient/interface/SiStripTrackerMapCreator.h

```diff
@@ -61,6 +61,7 @@ class SiStripTrackerMapCreator {
   float tkMapMin_;
   float meanToMaxFactor_;
   bool useSSQuality_;
+  bool ResidualsRMS;
```

Comment:
Hi @aspiezia - please follow the usual cms convention (and the one in this class) to have member data variables to end with '_'. Thanks